templates/navigation-bar-new.twig

```diff
@@ -147,4 +147,5 @@
 		</button>
 	</div>
 	<div class="top-navigation-overlay"></div>
+	{% include 'country_selector_banner.twig' ignore missing %}
```

Comment:
You are using underscores here for the template name, but dashes on the child theme 😁

Reply:
Very good point haha, I was confused as to why it didn't show while testing 🤦‍♀️

Reply:
I've updated the name in the child-theme, that made me realize that we also need to add there the svg for the close icon